arrow-ank-dokka-plugin/src/main/kotlin/dokka.kt

```diff
@@ -51,29 +51,31 @@ suspend fun List<DPackage>.parTraverseCodeBlock(
       node: DocumentationNode,
       wrapper: TagWrapper,
       CodeBlock) -> CodeBlock?
-): List<DPackage> = parTraverse { `package` ->
-  `package`.copy(
-    properties =
-      `package`.properties.map { property ->
-        property.copy(
-          documentation = property.documentation.process(dModule, `package`, property, transform)
-        )
-      },
-    functions =
-      `package`.functions.map { function ->
-        function.copy(
-          documentation = function.documentation.process(dModule, `package`, function, transform)
-        )
-      },
-    classlikes = `package`.classlikes.map { it.process(dModule, `package`, transform) },
-    typealiases =
-      `package`.typealiases.map { typeAlias ->
-        typeAlias.copy(
-          documentation = typeAlias.documentation.process(dModule, `package`, typeAlias, transform)
-        )
-      }
-  )
-}
+): List<DPackage> =
+  parTraverseN(1) { `package` ->
```

Comment:
```suggestion
  map { `package` ->
```

Let's also try avoiding async jumps.

Reply:
sorry I saw these before merging so I applied it later